### ISSUE TITLE
`Message too large` -> option to skip

### ISSUE THREAD

**Issue by itsrob** [joeyates/imap-backup#124]
Hey @joeyates, thanks so much for this great piece of software, this is really saving me!

I have one issue which is blocking me right now: I am migrating from a different email provider to Gmail and Gmail is rejecting an email for a too big attachment (I think):

```
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/net/imap.rb:1369:in `block in send_literal':  Message too large. https://support.google.com/mail/answer/6584#limit (Net::IMAP::BadResponseError)
```

Is there anyway to tell imap-backup to skip this email and continue or just skip any failing email? (I guess maybe something that would be helpful in general - could also have `n` retries).

Would really appreciate to get any tip from you how to solve this - and thanks again for working & maintaining imap-backup!

Best,
Robert

**Comment by joeyates** [comment]
Hi @itsrob 

I've created a branch with a fix.
If you have a chance to check it out, please do the following:

```sh
gem uninstall imap-backup # To uninstall the current version
git clone https://github.com/joeyates/imap-backup.git
cd imap-backup/
git checkout --track -b feature/handle-append-errors-in-migrator origin/feature/handle-append-errors-in-migrator
gem build imap-backup.gemspec
sudo gem install imap-backup-*.gem
```

**Comment by joeyates** [comment]
I've released the fix in version 6.2.0

**Comment by itsrob** [comment]
Thank you very much! That worked!
